README.md

```diff
@@ -62,16 +65,21 @@ cmake -DCMAKE_CURRENT_BINARY_DIR=$PWD \
 
 ### Install
 
+Once cmake configuration successfully completed, from the same build directory run build, install commands
+
 ```shell
-         make
-         make install
+cmake --build .
+cmake --install .
 ```
 
 ### Package Generated
 
+Once cmake configuration and build successfully completed from the same build directory trigger package generation
+
 ```shell
-         make package
+cpack
```

Comment:
umm. probably, but I would prefer "cmake --build . -- package".

Reply:
Updated the commands in the latest version.